src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -43,6 +61,23 @@
             />
           </div>
         </div>
+        <div class="onboarding-inputWrapper onboarding-inputWrapper--name"></div>
+        <div class="onboarding-inputWrapper onboarding-inputWrapper--name">
```

Comment:
most people shouldn't run into this but the wrapping for the drop-downs is a bit weird on a smaller screen. can we just have the rightmost items disappear if the screen is too small?
<img width="731" alt="image" src="https://user-images.githubusercontent.com/47431797/155941497-6d2ef1b9-0070-48dc-a24c-4b7e90120e5f.png">

Reply:
I agree it doesn't look great, but it looks pretty bad on my phone already, and I don't think just deleting the right column fixes the issue. Maybe talk to Kehui about any possible design updates to better improve this section? It was an item last sem but we never got around to it.

![image](https://user-images.githubusercontent.com/25535093/157145598-bc11feb2-1e49-436e-b22d-9b5fe16869cf.png)